seedlet/assets/sass/_variables.scss

```diff
@@ -1,6 +1,4 @@
-/**
- * These styles should be loaded on the frontend-only
- */
+// These styles should be loaded on the frontend-only
```

Comment:
I changed the style of comments here (`/**` to `//`) so they wouldn't be included in the compiled stylesheets.  
 Including the variables along with the comments made the beginning of the stylesheet look quite cluttered / difficult to read.